src/uk/gov/hmcts/contino/InfraPipelineDsl.groovy

```diff
@@ -11,4 +11,9 @@ class InfraPipelineDsl extends CommonPipelineDsl implements Serializable {
     this.steps = steps
   }
 
+  void expiresAfter(String expiresAfter) {
```

Comment:
should be in `CommonPipelineDsl` as you need it for both app and infra pipelines (app pipelines also do infra